js/Maps/Getting-Started.md

```diff
@@ -1,22 +1,22 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started with JavaScript Map Control | Syncfusion
```

Comment:
JavaScript `Maps` control, check and change in all places

Reply:
Modified